docs/api-docs/store-management/shipping/shipping-provider-api.mdx

```diff
@@ -37,13 +37,88 @@ BigCommerce makes a distinction between single-carrier and multi-carrier shippin
 
 ![Multi-carrier quote example](https://storage.googleapis.com/bigcommerce-production-dev-center/images/Multi%20Carrier%20Example.png)
 
+### Carrier configurations
+
+The configurations schema defines the optional settings that can be applied to enhance the user experience.
+
+If you're unsure about the configurations you may need, you can leave them unspecified for now and request to add them later.
```

Comment:
rather than `and request to add them later.` would it be better to say `and update then in a future PUT request.`
The former suggests they are requesting that we add them, the latter is an action on their behalf

Reply:
> The former suggests they are requesting that we add them, the latter is an action on their behalf.

This is exactly what I want it to mean. This is not an operation that they action themselves and has to be actioned by BigCommerce staff.